### DISCUSSION TITLE
Hazards in upwind method when dt is large

### DISCUSSION THREAD

**Discussion by Takodachi6969** [Part-II-Computational-Physics/.github#7] [General]
For any dt that is larger than 0.10 exactly there is a chance of a hazard appearing in the zero region of the graph (When it should be flat). The small hazard quickly explodes and causes the graph to overflow. Is it supposed to happen?

**Comment by Takodachi6969** [comment]
> For any dt that is larger than 0.10 exactly there is a chance of a hazard appearing in the zero region of the graph (When it should be flat). The small hazard quickly explodes and causes the graph to overflow. Is it supposed to happen?

The code was checked and the frames were inspected individually by plotting the graphs, it appeared that the notch occurred at the exact moment in time and space for the same dt values, where the computer thinks 0 minus 0 is some value, which explodes the entire thing...

**Comment by NathanMagnan** [comment]
I believe that this behaviour is expected, it might remind you of the part about stability of the lecture on ODEs.

**Reply by Takodachi6969** [reply]
Thank you, I was dumb when I wrote this issue...
